src/devices/vdml_motors.cpp

```diff
@@ -288,24 +685,47 @@ std::ostream& operator<<(std::ostream& os, const pros::Motor& motor) {
 	os << ", efficiency: " << motor.get_efficiency();
 	os << ", encoder units: " << (int)motor.get_encoder_units();
 	os << ", gearing: " << (int)motor.get_gearing();
-	os << ", over temp: " << motor.is_over_temp(); 
+	os << ", over temp: " << motor.is_over_temp();
 	os << ", position: " << motor.get_position();
 	os << ", reversed: " << motor.is_reversed();
 	os << ", temperature: " << motor.get_temperature();
 	os << ", torque: " << motor.get_torque();
-	os << ", voltage: " << motor.get_voltage(); 
+	os << ", voltage: " << motor.get_voltage();
 	os << "]";
 	return os;
 }
 
-void Motor::push_motor_configuration() const {
-	_motor_mutex.take();
+void Motor_Group::push_motor_configuration() const {
+	_motor_group_mutex.take();
 	set_gearing(_gearset);
 	set_reversed(_reverse);
 	set_encoder_units(_encoder_units);
-	_motor_mutex.give();
+	_motor_group_mutex.give();
```

Comment:
`std::lock_guard`

Reply:
I think we should stick with the pros mutexes since this is in the kernel's api.

Reply:
Why? `std::lock_guard` is just a RAII mutex wrapper, it's not an actual mutex. It does the exact same thing as calling `pros::Mutex::take()` and `::give()` but with less potential for mistakes.